src/components/Annotations/Ellipse.tsx

```diff
@@ -7,21 +7,21 @@ export interface AnnotationEllipseProps
     SVGProps<SVGEllipseElement>,
     'x1' | 'x2' | 'y1' | 'y2' | 'cx' | 'cy' | 'rx' | 'ry'
```

Comment:
I'm wondering if x1, x2, y1 and y2 really exists as props for an ellipse (same is true for a circle) and I don't know how to check this.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/SVGCircleElement
https://developer.mozilla.org/en-US/docs/Web/API/SVGEllipseElement

Reply:
I omitted x1, x2, etc because they exist in the TypeScript types so I thought it would help the user to remove them from intellisense.

Reply:
Ok then indeed we need to leave the code like it is now. Thanks.

Reply:
props omit fixed